source/common/http/conn_manager_utility.cc

```diff
@@ -212,7 +212,7 @@ void ConnectionManagerUtility::mutateTracingRequestHeader(Http::HeaderMap& reque
   }
 
   if (!runtime.snapshot().featureEnabled("tracing.global_enabled",
-                                         config.tracingConfig()->overall_sampling_, result)) {
+                                         config.tracingConfig()->overall_sampling_, result, 100)) {
```

Comment:
why this change?

Reply:
See [this comment](
https://github.com/envoyproxy/envoy/pull/3978#discussion_r208368275)